code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -83,6 +83,30 @@
 		if("cry")
 			message = "<B>[comm_paygrade][src]</B> cries."
 
+		if("dab")
+			message = "<B>[comm_paygrade][src]</B> dabs."
+			log_message(message, LOG_EMOTE)
+
+			for(var/mob/M in dead_mob_list)
+				if(!M.client || istype(M, /mob/new_player))
+					continue
+				if(M.stat == DEAD && M.client.prefs && (M.client.prefs.toggles_chat & CHAT_GHOSTSIGHT) && !(M in viewers(src,null)))
+					M.show_message(message, m_type)
+
+			if(m_type == EMOTE_VISIBLE)
+				for(var/mob/O in viewers(src, null))
+					O.show_message(message, m_type)
+			else if(m_type == EMOTE_AUDIBLE)
+				for(var/mob/O in hearers(loc, null))
+					O.show_message(message, m_type)
+			var/datum/limb/l_arm/A = get_limb("l_arm")
+			if(!(A.status & LIMB_DESTROYED))
+				if(!(A.status & LIMB_DESTROYED)) A.droplimb()
```

Comment:
Ditto, also it would end up removing both arms, which is unnecessary evil for most.